### PR TITLE
Our-journey timeline proposal

### PR DESCRIPTION
**Pull request by fahimaqil**
Added timeline to the http://localhost:3000/lapor-predator route.

Please review especially on the:
1) Ui/Ux part
2) The arrangement of the .md file. I made a new folder which is our-journeys
3) Vue technical implementation

### CODE REVIEW

File: components/ui/Timeline.vue [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<template>
```
Comment (knazran) [inline components/ui/Timeline.vue]: Remove this component

File: pages/lapor-predator/index.vue [hunk 2]
```diff
@@ -33,6 +33,29 @@
         />
       </div>
     </PageSection>
+    <PageSection id="timeline">
+      <PageTitleAlt title="Our Journey" />
+      <div class="relative wrap overflow-hidden p-10 h-full">
```
Comment (knazran) [inline pages/lapor-predator/index.vue]: Padding is a little too big on mobile. Reduce it to px-4 or px-6

File: components/ui/LeftTimeline.vue [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        items-center
+        order-1
+        bg-gray-800
+        shadow-xl
+        w-36
+        h-36
+        rounded-full
+      "
+    >
+      <img
```
Comment (knazran) [inline components/ui/LeftTimeline.vue]: Can we have the image inside the card itself? Else its too small

Comment (fahimaqil) [reply components/ui/LeftTimeline.vue]: Dah edit, can you check again

**Comment by knazran** [comment]
LGTM.

**Pull request merged**